node/src/test/java/io/airlift/node/TestNodeConfig.java

```diff
@@ -46,12 +49,20 @@ public void testDefaults()
                 .setLocation(null)
                 .setBinarySpec(null)
                 .setConfigSpec(null)
-                .setInternalAddressSource(IP));
+                .setInternalAddressSource(IP)
+                .setAnnotationFile(null));
     }
 
     @Test
     public void testExplicitPropertyMappings()
     {
+        File annotationFile;
```

Comment:
You can use `TempFile` from the `testing` module which supports try-with-resources

Reply:
Actually, we could reuse the `annotations.properties` testing resource file here